robottelo/cli/product.py

```diff
@@ -55,3 +55,15 @@ def set_sync_plan(cls, options=None):
         result = cls.execute(cls._construct_command(options))
 
         return result
+
+    @classmethod
+    def synchronize(cls, options=None):
+        """
+        Synchronize a product.
+        """
+
```

Comment:
All blank lines here could be dropped. Don't worry about other cli parts should be updated when touched.